xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-wiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -159,7 +159,7 @@ WikiManager.WikiCandidateMemberClass_type_invitation=Invitation
 # HINTS
 platform.wiki.currentwiki.prop.wikiprettyname.hint=The display name of the wiki.
 platform.wiki.currentwiki.prop.description.hint=A brief description of the wiki and its purpose.
-platform.wiki.currentwiki.prop.owner.hint=A user who will have all the rights on the wiki.
+platform.wiki.currentwiki.prop.owner.hint=A user who has access to all the rights on the wiki.
```

Comment:
You "have a right" or you don't but "having access to a right" does not seems correct to me.